Makefile

```diff
@@ -79,6 +79,9 @@ cross: build ## cross build the binaries for darwin, freebsd and\nwindows
 win: build ## cross build the binary for windows
 	$(DOCKER_RUN_DOCKER) hack/make.sh win
 
+darwin: build ## cross build the binary for OS X
+	$(DOCKER_RUN_DOCKER) hack/make.sh darwin
```

Comment:
Where did that `win` target come from??  :tired_face:

(for the trick you're suggesting to work, which I like, `DOCKER_CROSSPLATFORMS` will have to be injected via `-e` on `docker run` before the image, or we'll have to use a command like `sh -c 'DOCKER_CROSSPLATFORMS="darwin/amd64" hack/make.sh cross'` instead)